lib/function/arithmetic/log10.js

```diff
@@ -31,7 +31,9 @@ function factory (type, config, load, typed) {
   var log10 = typed('log10', {
     'number': function (x) {
       if (x >= 0 || config.predictable) {
-        return Math.log(x) / Math.LN10;
+        return (Math.log10)
```

Comment:
Thinking about it: the JavaScript engines will probably optimize this themselves. We could do a simple benchmark test to see if it improves. If not, I prefer the way you implemented it right now, it's better readable.